karma.conf.js

```diff
@@ -61,40 +62,16 @@ var localLaunchers = {
 	}
 };
 
-const babelOptions = (options = {}) => {
-	return {
-		babelrc: false,
-		cacheDirectory: true,
-		presets: [
-			[
-				'@babel/preset-env',
-				{
-					loose: true,
-					exclude: ['@babel/plugin-transform-typeof-symbol'],
-					targets: {
-						browsers: ['last 2 versions', 'IE >= 9']
-					}
-				}
-			]
-		],
-		plugins: [
-			coverage && ['istanbul', { include: '**/src/**/*.js' }],
-			'@babel/plugin-proposal-object-rest-spread',
-			options.debug && '@babel/plugin-transform-react-jsx-source',
-			'@babel/plugin-transform-react-jsx',
-			'babel-plugin-transform-async-to-promises'
-		].filter(Boolean),
-		ignore: ['./dist']
-	};
-};
+const subPkgPath = pkgName =>
+	path.join(__dirname, pkgName, !minify ? 'src' : '');
```

Comment:
Selectively load the minified files or the src files depending on the MINIFY env variable